docs/User/ProjectRelated/Changelog.md

```diff
@@ -3,60 +3,79 @@
 <!-- See rendered changelog at https://streamly.composewell.com -->
 ## 0.9.0 (Unreleased)
 
-### Major Changes
+### Package split
 
 `streamly` package is split into two packages, (1) `streamly-core` that
 has only GHC boot library depdendecies, and (2) `streamly` that contains
-higher level operations with additional dependencies.
-
-### Breaking Changes
+higher level operations (including concurrent ones) with additional
+dependencies. Make sure you add a dependency on `streamly-core` to keep old
+code working.
 
-* Moved the following modules to the `streamly-core` package:
+* Moved the following modules from `streamly` package to the
+  `streamly-core` package:
   * Streamly.Console.Stdio
   * Streamly.Data.Fold
   * Streamly.Data.Unfold
   * Streamly.FileSystem.Handle
   * Streamly.Unicode.Stream
-* The unboxed arrays now require `Unbox` instead of `Storable` for the stored
-  type. The `Unbox` typeclass can be found in `Streamly.Data.Array`.
-* Remove the MonadBase instance of the SerialT type.
-* In `Streamly.Data.Unfold`, signatures changed:
+
+### Breaking Changes
+
+* Unboxed arrays now require `Unbox` constraint instead of `Storable`.
+  The `Unbox` typeclass can be imported from `Streamly.Data.Array`. You
+  can use generic deriving to derive Unbox instances.
+* Stream type in all modules has been changed to the new `Stream` type
+  replacing the existing `IsStream t` or `SerialT` types. Use `fromStream`,
+  `toStream` from `Streamly.Prelude` module to adapt the types.
+* Signatures changed in `Streamly.Data.Unfold`:
   * `fromStream`
   * `replicateM`
 
+### Major Changes
+
+`Streamly.Prelude` module has been deprecated, equivalent
+functionality is covered by the `Streamly.Data.Stream`,
```

Comment:
`Streamly.Data.Stream` is re-exported from `Streamly.Data.Stream.Prelude` right?

Reply:
Yes, it is.